.changes/unreleased/tinymce-TINY-11676-2025-04-17.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Now pressing enter before a floating wrapped image doesn't cause to duplicate that image.
```

Always past tense for bug fixes please. I wonder if we can train an AI to verify this for us 😂 

```suggestion
body: Pressing enter before a floating image sometimes duplicated the image.
```

---

I'll try to generate it via AI next time, maybe with all the changelog history it's able to do a better job :sweat_smile: